src/app/calendar/clientPage.tsx

```diff
@@ -0,0 +1,41 @@
+"use client";
+
+import React from "react";
+import Calendar, { CalendarEvent as CalendarEvent } from "../../components/Calendar/Calendar";
+import { styled } from "styled-components";
+
+const SampleCalendar: React.FC<{}> = () => {
+    const sampleEvents: CalendarEvent[] = [
```

Comment:
Are "All Day" events only implicit upon omitting the times from the `start` and `end` props? I know this is probably how the library works, but it would be nice if all day events were declared explicitly by a boolean `allDay` prop.

Reply:
we have added an optional allDay props.

Reply:
if allDay is set to True but the start or end has time, the time will be ignored.